app/models/user.rb

```diff
@@ -7,7 +7,7 @@
 #  id                  :bigint           not null, primary key
 #  current_sign_in_at  :datetime
 #  current_sign_in_ip  :string
-#  email               :string           default(""), not null
+#  email               :string           default("")
```

Comment:
I'm wondering why we've got a `default("")` if we're allowing the column to be `null`? We could leave the emails as blank for these users rather than `null`.

Reply:
Ya, I was _just_ thinking the same thing. Will change.

Reply:
Changed